time_tracker_api/time_entries/time_entries_model.py

```diff
@@ -116,21 +125,30 @@ def find_all(
         conditions: dict = {},
         date_range: dict = {},
     ):
-        custom_sql_conditions = []
-        custom_sql_conditions.append(
-            self.create_sql_date_range_filter(date_range)
-        )
+        custom_sql_conditions = [self.create_sql_date_range_filter(date_range)]
 
-        custom_params = self.generate_params(date_range)
+        if event_context.is_admin:
```

Comment:
Can we move this validation to the DAO?

Reply:
If so, we could check if user is admin and obviate `conditions.update({"owner_id": event_ctx.user_id})` in line 305